src/lang/export.pl

```diff
@@ -44,3 +44,23 @@
   tripledb_export(Directory),
   obda_export(Directory),
   forall(memorize_hook(Directory), true).
+
+%      /*******************************
+%      *          UNIT TESTS          *
+%      *******************************/
+
+% :- begin_tests('lang_export').
```

Comment:
could you try adding *blocked(+Reason:atom)* option? then you might not need to comment the code below.

Reply:
Yes